en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+
+- **95th Percentile Latency**
+
+    ![95th percentile latency](../assets/img/insights/overview-page-error-rate.png){.cInlineImage-half}
+
+    This widget displays the 95th percentile of all API latencies in your selected environment for the selected time interval. You can use this widget to know whether the whole system operates under given SLAs. This metric gives the first indication of slow APIs. To investigate further, see Viewing API Analytics on Latency. 
+
+- **API Request Summary**
+
+    ![API request summary](../assets/img/insights/overview-page-timeline.png){.cInlineImage-full}
+
+    This chart combines the above three widgets and plots them in a timeline. The y-axis on the left displays the request count and the error count. The x-axis shows time, and the y-axis on the right shows the latency in milliseconds. The granularity of the data points is decided based on the time range you have selected. The tooltip provides the exact value of all three metrics accurately.
+
+### Traffic
+
+This page shows information related to the traffic that goes through your API management system. This includes API usage, application usage, resource usage, etc. You can use this page to investigate the usage of APIs and applications, traffic patterns, etc.
+
+![Traffic](../assets/img/insights/traffic-page-full.png){.cInlineImage-full}
+
+You can filter the information displayed in the widgets as follows:
+
+| **Filtering Option** | **Description** |
+|--------------------|-------------------------------------------------------------------------------------------------|
+| **By API**         | In the **API** field, you can select the APIs for which you want to view analytics. **All** is selected by default. You can select one or multiple APIs from this selector and view the aggregated result. Once you select an API, you can further filter by a specific application that uses the selected API via the **Application** field described below. |
+| **By Application** | In the **Application** field, you can select the applications for which you want to view analytics. **All** is selected by default. The available applications are all the applications that have subscribed to one or more of the APIs you selected in the **API** field.|
+
+The information you can view is as follows:
+
+ - **API Usage Over Time**
+ 
+    This timeline shows the count of API hits for the APIs you selected using the above-mentioned selectors. If multiple APIs are selected, the timeline shows each API in a separate line with a legend separating each line. You can also zoom in on a selected time range by selecting that area in the chart. To restore to the original view use the **Zoom out** button on the top right-hand corner of the plot.
```

Comment:
This timeline shows the count of API hits for the filtered APIs.

Reply:
Done